content/blog/2019-02-01-software-peer-review-news.md

```diff
@@ -0,0 +1,101 @@
+---
+slug: software-review-news
+title: 'rOpenSci Software Peer Review: Still Improving'
+date: 2019-02-01
+authors:
+  - Maëlle Salmon
+  - Brooke Anderson
+  - Scott Chamberlain
+  - Anna Krystalli
+  - Lincoln Mullen
+  - Karthik Ram
+  - Noam Ross
+  - Melina Vidoni
+categories: blog
+topicid: 
+tags:
+  - Software Peer Review
+---
+
+rOpenSci's [suite of packages](/packages/) is comprised of contributions from staff engineers and the wider R community, bringing considerable diversity of skills, expertise and experience to bear on the suite. How do we ensure that every package is held to a high standard? That's where our software review system comes into play: packages contributed by the community undergo a **transparent, constructive, non adversarial and open review process**. For that process relying mostly on **volunteer work**, _[associate editors](/about#team)_ manage the incoming flow and ensure progress of submissions; _authors_ create, submit and improve their package; *[reviewers](https://ropensci.github.io/dev_guide/#reviewers)*, two per submission, examine the software code and user experience.
```

Comment:
For this link  `[associate editors](/about#team)` would it be better to link to dev guide like `[associate editors](https://ropensci.github.io/dev_guide/softwarereviewintro.html#associateditors)` to get more full info? 

about/#team editors info is hidden below first screen

Reply:
ok!